src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt

```diff
@@ -414,6 +415,27 @@ object ItemFeatures {
                 }"
             })
         }
+
+        if (Skytils.config.showItemQuality && extraAttr != null) {
+            val boost = extraAttr.getInteger("baseStatBoostPercentage")
+            val tier = extraAttr.getInteger("item_tier")
+
+            if (boost > 0 && tier > 0) {
+                val isMasterMode = requirementPattern.matchEntire(extraAttr.getString("dungeon_skill_req"))?.let {
+                    val level = it.groups["level"]?.value?.toIntOrNull() ?: return@let null
+                    level > 24
+                }
+
+                val floor = when (isMasterMode) {
+                    true -> "§4M${tier - 3}"
+                    false -> "§aF$tier"
+                    null -> "§b$tier"
+                }
+
+                event.toolTip.add("§6Quality: §b$boost% §7($floor§7)")
```

Comment:
Maybe we could associate colors to certain quality ranges? (i.e going from red to green, 0-16% red, 17-33% yellow, 34-49% green, 50% aqua)

Reply:
aqua as the best seems really weird, maybe red-orange-yellow-green would be better (although i dont think people care about lower qualities than 50%?)

Reply:
I just used aqua as an example because it's what's currently used. Though having a visually distinct color for best would probably be best as it allows for easier differentiation at a glance.

Reply:
I feel like aqua fill fit nicely with the other tooltips added at the bottom so I'll use that as the top quality